data_generator/util_compare_dicts.py

```diff
@@ -0,0 +1,46 @@
+import collections
+
+
+def flatten(d, sep="_"):
+    obj = collections.OrderedDict()
```

Comment:
Why not use a normal dict?

Reply:
Same as before -- superstition about the ordering of keys. Fixed!